lib/iris/tests/integration/test_netcdf.py

```diff
@@ -592,5 +594,79 @@ def test_standard_name_roundtrip(self):
             self.assertEqual(detection_limit_cube.standard_name, standard_name)
 
 
+class TestLoadMinimalGeostationary(tests.IrisTest):
+    """
+    Check we can load data with a geostationary grid-mapping, even when the
+    'false-easting' and 'false_northing' properties are missing.
+
+    """
+
+    _geostationary_problem_cdl = """
```

Comment:
If this is from real data, I suggest we should be storing the original in [iris-test-data](https://github.com/SciTools/iris-test-data) and using that. If this is a highly simplified test case then we should store it in a dedicated file within [/tests/results/integration/](lib/iris/tests/results/integration), as has been done with the other use of ncgen in testing - [/tests/integration/test_climatology.py](lib/iris/tests/integration/test_climatology.py).

Reply:
> If this is a highly simplified test case then we should store it in a dedicated file within /tests/results/integration/, as has been done with the other use of ncgen in testing - /tests/integration/test_climatology.py.

Well, it _is_ such a "highly simplified" testcase, I think.
But I _don't_ think it would be better to have the CDL in a file : I think it's simpler to have it here where you can read it, right in the only code that uses it.
In the 'test_climatology' case, I think we wanted an actual file under 'results', just so we could use `IrisTest.assertCDL` to check a round-tripped result against the original.

Reply:
I agree with the principle. This isn't something that's been done before in Iris but hopefully it's a precedent we can establish while working on #3629 .